config/routes.js

```diff
@@ -171,15 +171,6 @@ module.exports = [
     },
   },
 
-  {
-    method: 'GET',
-    path: '/api/v2/updatelistusers',
-    handler: ListUserController.updateListUsers,
-    config: {
-      auth: false,
-    },
-  },
-
```

Comment:
I know we already discussed this, so it's not a request for a modification per se, but I want to go on record saying it seems very unnecessary to mix removals of API code with other PRs (even if you're personally quite confident it won't have an effect.. we all make mistakes sometimes).

If someone's using this, we just very unceremoniously slipped a breaking change into a commit that doesn't even announce the change. it just says "improved logging" on most commits in this PR, with two mentions of "and removed code"

I'd personally isolate API removals in their own commit, PR, and release.

Removing an API method without typically implies a new major version for the software. I assume this removal is part of the deprecated code mentioned in the commits, but do we have usage stats that guarantee it is unused at the moment? I would think this logging PR will give us that visibility, but until it's live we can't collect that data so easily.

Reply:
This is not an API method. 
This is a callback which I used at a much earlier point before switching to nodejs commands via Jenkins, which has now been moved to a command. I agree that it would be better if it were in its own commit, but I'm sure no one is using this (and even if someone is, this is not code that should be used anymore anyways).

Reply:
Ok, I can accept that. Maybe in the future we can also document deprecated things and schedule their removal, or at least mark them as needing to be removed at some point. 

Lacking clear documentation seems to be the main problem here, causing me to be overly cautious.